app/Main.hs

```diff
@@ -1,35 +1,59 @@
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE DuplicateRecordFields #-}
 
 module Main where
 
 import Data.Aeson
 import qualified Data.Text as Text
 import qualified Data.Text.Encoding as Text
+import qualified Data.List as List
+import qualified Data.Map as Map
+import qualified Data.Set as Set
 import Data.Version (showVersion)
 import qualified Paths_spacchetti_cli as Pcli
+import qualified System.Directory as Dir
 import qualified Templates
 import qualified Turtle as T
 import qualified Types
 
 -- | Commands that this program handles
 data Command
+  -- | # Commands for working with Psc-Package:
+
   -- | Do the boilerplate of the local project setup to override and add arbitrary packages
   -- | See the Spacchetti docs about this here: https://spacchetti.readthedocs.io/en/latest/local-setup.html
   = LocalSetup Bool
   -- | Do the Ins-Dhall-ation of the local project setup, equivalent to:
   -- | ```sh
   -- | NAME='local'
   -- | TARGET=.psc-package/$NAME/.set/packages.json
-  -- | mkdir -p .psc-package/$NAME/.set
+  -- | mktree -p .psc-package/$NAME/.set
   -- | dhall-to-json --pretty <<< './packages.dhall' > $TARGET
   -- | echo wrote packages.json to $TARGET
   -- | ```
   | InsDhall
```

Comment:
How about renaming this (or aliasing if you want to keep backwards compatibility for a while) to `psc-package-install`?

Reply:
This was a mistake with iedit